assets/javascripts/discourse/components/chat-message-collapser.js

```diff
@@ -105,6 +109,25 @@ export default class ChatMessageCollapser extends Component {
       return acc;
     }, []);
   }
+
+  galleryCooked() {
+    const elements = Array.prototype.slice.call(domFromString(this.cooked));
+
+    return elements.reduce((acc, e) => {
+      if (galleryPredicate(e)) {
+        const link = e.firstElementChild.href;
+        const title = e.firstElementChild.firstElementChild.textContent;
+        e.firstElementChild.removeChild(e.firstElementChild.firstElementChild);
```

Comment:
Deletes this overlay by our oneboxer:

<img width="626" alt="Screenshot 2022-05-10 at 12 16 01 AM" src="https://user-images.githubusercontent.com/1555215/167452990-cedb3c7b-06d5-478b-a5e7-151af6b9b5de.png">